.ruby-version

```diff
@@ -1 +1 @@
-2.5.3
+2.5.5
```

Comment:
Ruby 2.5 is technically no longer supported, I think. Do we want to bump this up to at least 2.6.x? 🤷